nonio-ios/Sources/Common/NotificationUnreadTicker.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import Combine
+import Moya
+
+class NotificationUnreadTicker: ObservableObject {
+    @Published private(set) var unreadCount: Int = 0
+
+    private let provider = MoyaProvider.defaultProvider
+    private var cancellables = Set<AnyCancellable>()
+    private let tickerInterval: TimeInterval = 20
```

Comment:
Is this a fixed ticker interval? Can we change it to be an exponential decay that resets every time the app is opened? I want to make sure that browsing doesn't lead to a DDOS. 

You can see some code for the frontend for this here: https://github.com/jjcm/nonio-frontend/blob/master/components/soci-notification-badge.js#L94

Would love if you could make it have this logic:

- Start with the interval every 10s
- Every unsuccesful check, set the interval to the previous interval * 1.3, to a max of 30min
- Set the interval duration to back to 10s on the following events:
  - Creating a post or comment
  - Logging in
  - Opening the app
  - Receiving a new notification

Reply:
Hi @jjcm , this is fixed in 7d50f2fc85b75e59abe85c301d23df1964654f91